### PR TITLE
Translate all labels and placeholders in seed form to english.

### PR DESCRIPTION
**Pull request by badnames**
## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Review by buenaflor — APPROVED** [review]

File: frontend/src/features/seeds/components/CreateSeedForm.tsx [hunk 1]
```diff
@@ -110,16 +110,16 @@ const CreateSeedForm = ({ onCancel, onChange, onSubmit }: CreateSeedFormProps) =
...
             id="origin"
             register={register}
             onChange={onChange}
           />
           <SimpleFormInput
             type="date"
-            labelText="Verbrauch bis"
-            placeHolder="Verbrauch bis"
+            labelText="Best by"
+            placeHolder="01.01.2000"
```
Comment (markus2330) [inline frontend/src/features/seeds/components/CreateSeedForm.tsx:122]: Can you add the current year here?

**Review by markus2330 — APPROVED** [review]
Great job with the translations.
It is a bit of a pity, however, that the translations would get lost. Can you add these translations in an react i18next file (or simply in a text file so that we have them later when we translate.)

**Comment by badnames** [comment]
> Great job with the translations. It is a bit of a pity, however, that the translations would get lost. Can you add these translations in an react i18next file (or simply in a text file so that we have them later when we translate.)

They will still be in the git history, although I agree that having a separate file is more convenient.

File: frontend/src/features/seeds/components/CreateSeedForm.tsx (hunk 1 above)
Comment (badnames) [reply frontend/src/features/seeds/components/CreateSeedForm.tsx:122]: Yes of course.

**Comment by markus2330** [comment]
@badnames please don't self-merge unless there is an important reason to do so, especially not if there were open tasks (like extracting the translations). Furthermore, the branch was not removed.

I created  #150 so that we don't forget where the relevant PRs/commits for translation were.

**Pull request merged**